### ISSUE TITLE
[KNX2] wrong value for DPT 14.017

### ISSUE THREAD

**Issue by pdcemulator** [openhab/openhab2-addons#3409]
Hello,

I am using items with DPT 14.017 (density in g/m³).

## Expected Behavior
value of 7.1 g/m³ is shown as 7.1 g/m³

## Current Behavior
value of 7.1 g/m³ is shown as 570.9 g/m³

## Possible Solution
?

## Steps to Reproduce (for Bugs)
In OH1 I used
`Number Hum_abs "Luftfeuchte abs. [%.1f g/m³]"<humidity> (gHumidity) {knx="14.001:5/2/104"}`

In OH2
`Type number : Hum_abs "Humidity" [ ga="14.001<5/2/104" ]`
`Number Hum_abs "Luftfeuchte abs. [%.1f g/m³]"<humidity> (gHumidity) {channel="knx:device:bridge:wg:Hum_abs}`

DPT 14.017 instead of 14.001 in channel config give same value.

ETS capture
![capture](https://user-images.githubusercontent.com/20071350/38031202-f78dbd92-329a-11e8-82ee-3b70d1140565.PNG)

## Your Environment
* OH2.3 Snapshot on Synology NAS (Docker 2.3.0-snapshot-amd64-alpine)
* KNX IP Gateway Weinzierl 730
* Wiregate 1wire MultiSensors

**Comment by sjka** [comment]
For sure @lewie is the expert here, but to me it looks like this will be covered by openhab/openhab1-addons#5277, right? 

As a side-note: I personally would prefer to keep the `KNXCoreTypeMapper` class identical in both bindings identical as long as possible, therefore I would simply wait until openhab/openhab1-addons#5277 got merged into KNX1 and then take it over into the KNX2 binding here. If we do it in parallel in both bindings we're gonna end up in a big mess, I fear...

**Comment by lewie** [comment]
@sjka, DPT 14.017 is correctly mapped to `DecimalType.class` in KNXCoreTypeMapper.java

Synced [KNXCoreTypeMapper ](https://github.com/openhab/openhab1-addons/pull/5277/commits/be8298657be87e443243ac7724dbaef55f70cb4e)

**Comment by lewie** [comment]
@pdcemulator, the correct syntax is:
`Type number : Hum_abs "Humidity" [ga="14.017:<5/2/104"]` 

the colon is crucial. Then it works as expected and I'll get back the correct `7.12` value.

**Comment by pdcemulator** [comment]
Thank you, this must be a typo while creating the issue. I tested with colon, now again but it does not work. But after a restart of OH it works now. Thank you so much!
